### PR TITLE
HPCC-16900 Add multi-line string constants to ECL

### PR DESCRIPTION
**Pull request by richardkchapman**
Signed-off-by: Richard Chapman <rchapman@hpccsystems.com>

### CODE REVIEW

**Comment by richardkchapman** [comment]
@ghalliday Please review

File: ecl/hql/hqllex.l [hunk 1]
```diff
@@ -1548,6 +1549,64 @@ FUNCTIONMACRO|MACRO {
...
+                        int startpos = returnToken.pos.position;
+                        if (lexer->yyBuffer[startpos] == 'u')
+                        {
+                            bool isUtf8 = false;
+                            startpos++;
+                            if (lexer->yyBuffer[startpos]=='8')
+                            {
+                                isUtf8 = true;
+                                startpos++;
+                                }
```
Comment (ghalliday) [inline ecl/hql/hqllex.l]: trivial: indentation.

**Comment by ghalliday** [comment]
@richardkchapman trivial comment

**Comment by richardkchapman** [comment]
@ghalliday Repushed to fix indentation

**Pull request merged**